fpmsyncd/routesync.cpp

```diff
@@ -45,6 +47,65 @@ using namespace swss;
 
 #define ETHER_ADDR_STRLEN (3*ETH_ALEN)
 
+#define DEFAULT_SRV6_MY_SID_BLOCK_LEN "32"
+#define DEFAULT_SRV6_MY_SID_NODE_LEN "16"
+#define DEFAULT_SRV6_MY_SID_FUNC_LEN "16"
+#define DEFAULT_SRV6_MY_SID_ARG_LEN "0"
+
+enum srv6_localsid_action {
+	SRV6_LOCALSID_ACTION_UNSPEC				= 0,
+	SRV6_LOCALSID_ACTION_END				= 1,
+	SRV6_LOCALSID_ACTION_END_X				= 2,
+	SRV6_LOCALSID_ACTION_END_T				= 3,
+	SRV6_LOCALSID_ACTION_END_DX2			= 4,
```

Comment:
Current SAI doesn't support L2 XConnect, does FRR support this?

Reply:
No, currently FRR does not support L2 XConnect.